utils/wasm-builder/src/lib.rs

```diff
@@ -35,8 +35,19 @@ mod crate_info;
 mod multiple_crate_versions;
 mod smart_fs;
 mod wasm_project;
+pub use constants::*;
 
-pub const TARGET: &str = env!("TARGET");
+/// Module containing constants of wasm-builder
+pub mod constants {
+    pub const TARGET: &str = env!("TARGET");
+
+    /* CodeValidator related constants. */
+    pub(crate) const STACK_HEIGHT_LIMIT: u32 = 36_743;
+
+    pub(crate) const DATA_SEGMENTS_AMOUNT_LIMIT: u32 = 1024;
+
+    pub(crate) const TABLE_NUMBER_LIMIT: u32 = 100;
+}
 
```

Comment:
plz use `let schedule = gear_core::gas_metering::Schedule::default()` instead of this hardcoded value
https://github.com/gear-tech/gear/issues/4462#issuecomment-2607206078

Reply:
https://github.com/gear-tech/gear/blob/caf74e79638c16b255426a0c9f505b778337048e/core/src/gas_metering/schedule.rs#L149
https://github.com/gear-tech/gear/blob/caf74e79638c16b255426a0c9f505b778337048e/core/src/gas_metering/schedule.rs#L155
https://github.com/gear-tech/gear/blob/caf74e79638c16b255426a0c9f505b778337048e/core/src/gas_metering/schedule.rs#L161